### PR TITLE
Feature/Landingpage

### PR DESCRIPTION
**Pull request by IrisHerholz**

### CODE REVIEW

File: componentens/Landingpage/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <>
+      <container>
+        {" "}
+        <Form onSubmit={handleSubmit}>
+          <p>Where is the journey going?</p>
+          <Label htmlFor="name">City or Country?</Label>
+          <input
+            type="text"
+            required
+            id="entry"
```
Comment (ThomasF85) [inline componentens/Landingpage/index.js]: The `htmlFor` attribute is connected to the `id` of the input. You have to use the same value for both in order for the input to be accessible.

File: componentens/Landingpage/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import styled from "styled-components";
+
+export default function Landingpage() {
+  const [entry, setEntry] = useState("");
+  const [message, setMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    setMessage(`${entry}`);
```
Comment (ThomasF85) [inline componentens/Landingpage/index.js]: Unnecessary interpolation, you can write it like this: `setMessage(entry)`

File: componentens/Landingpage/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            type="text"
+            required
+            id="entry"
+            name="entry"
+            value={entry}
+            placeholder="enter your destination..."
+            onChange={(event) => setEntry(event.target.value)}
+          />
+          <Button type="submit">Submit</Button>
+          <>{message}</>
```
Comment (ThomasF85) [inline componentens/Landingpage/index.js]: You should put the string `message` into an html element (not into a fragment)

File: componentens/Landingpage/index.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  border-radius: 4px;
+  font-size: 1.5em;
+  text-align: center;
+`;
+
+const Button = styled.button`
+  border-radius: 3px;
+  margin: 0.5em 1em;
+  padding: 0.25em 1em;
+`;
```
Comment (ThomasF85) [inline componentens/Landingpage/index.js]: You are mixing relative and absolute units for similar properties. margin, padding, border-radius should all have consistent units.

File: componentens/Landingpage/index.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import React, { use } from "react";
```
Comment (ThomasF85) [inline componentens/Landingpage/index.js]: your folder name has a typo

File: pages/index.js [hunk 6]
```diff
@@ -1,3 +1,17 @@
+import Landingpage from "@/componentens/Landingpage";
+import styled from "styled-components";
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <>
+      {" "}
+      <Title>Your Travel Journal App</Title>
+      <Landingpage />
```
Comment (ThomasF85) [inline pages/index.js]: I recommend not turning the Landingpage into a component and instead copying all content from the component into this page component

File: componentens/Landingpage/index.js [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+    setMessage(`${entry}`);
+    setEntry("");
+  };
+  return (
+    <>
+      <container>
+        {" "}
+        <Form onSubmit={handleSubmit}>
+          <p>Where is the journey going?</p>
```
Comment (ThomasF85) [inline componentens/Landingpage/index.js]: This could act as the label of the form (via aria-labelledby)

**Review by ThomasF85 — CHANGES_REQUESTED** [review]
Please check my comments

File: componentens/Landingpage/index.js (hunk 4 above)
Comment (doemser) [reply componentens/Landingpage/index.js]: margin, padding and border-radius still have mixed units

File: pages/index.js [hunk 8]
```diff
@@ -1,3 +1,70 @@
...
+          <Label htmlFor="name">City or Country?</Label>
+          <input
+            type="text"
+            required
+            id="text"
+            name="entry"
+            value={entry}
+            placeholder="enter your destination..."
+            onChange={(event) => setEntry(event.target.value)}
+          />
```
Comment (doemser) [inline pages/index.js]: htmlFor needs to have the same value as id

File: componentens/Landingpage/index.js (hunk 7 above)
Comment (doemser) [reply componentens/Landingpage/index.js]: this was not added as well

**Comment by IrisHerholz** [comment]
i sent the wrong PR, sorry

File: pages/index.js [hunk 9]
```diff
@@ -1,3 +1,70 @@
...
+    <>
+      <container>
+        <Title>Your Travel Journal App</Title>
+        <Form
+          onSubmit={handleSubmit}
+          aria-labelledby="Where is yout jouney going?"
+        >
+          <p>Where is the journey going?</p>
+
+          <Label htmlFor="name">City or Country?</Label>
```
Comment (doemser) [inline pages/index.js]:
```suggestion
        <Form
          onSubmit={handleSubmit}
          aria-labelledby="form-heading"
        >
          <p id="form-heading">Where is the journey going?</p>

          <Label htmlFor="name">City or Country?</Label>
```

**Review by doemser — CHANGES_REQUESTED** [review]

File: pages/index.js [hunk 10]
```diff
@@ -1,3 +1,67 @@
...
+        >
+          <Label htmlFor="text">City or Country?</Label>
+          <input
+            type="text"
+            required
+            id="text"
+            name="entry"
+            value={entry}
+            placeholder="enter your destination..."
+            onChange={(event) => setEntry(event.target.value)}
```
Comment (F-Kirchhoff) [inline pages/index.js]: Do you need to control your input? I can also use the "formData" approach in your handleSubmit instead of using the entry state.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Iris, this is looking solid! I just wonder if you need to control your input? You could get rid of your entry state.

File: pages/index.js (hunk 10 above)
Comment (IrisHerholz) [reply pages/index.js]: Hey Felix!
I dont need to control the input for now. I just want that the entry is displayed and that was the first way i thought of implementing it. Can I leave it like this? Or is the "formData" approach recommended?

Comment (F-Kirchhoff) [reply pages/index.js]: Then I would reccommend the formData approach, then you don't have to juggle around with the two states. You could also just use `event.target.elements.<name of input>.value`

File: pages/index.js [hunk 11]
```diff
@@ -1,15 +1,13 @@
-import { useState } from "react";
 import styled from "styled-components";
+import React from "react";
```
Comment (F-Kirchhoff) [inline pages/index.js:2]: you don't need this React import here.

File: pages/index.js [hunk 12]
```diff
@@ -1,15 +1,13 @@
...
-
+  const [output, setOutput] = useState("");
   const handleSubmit = (event) => {
     event.preventDefault();
-
-    setMessage(entry);
-    setEntry("");
+    const entry = event.target.elements.text.value;
+    setOutput(entry);
   };
```
Comment (F-Kirchhoff) [inline pages/index.js:11]: nice!

**Review by F-Kirchhoff — APPROVED** [review]
Please remove the React import that isn't needed. Otherwise: LGTM 🦊

**Pull request merged**